2015/10-dancer-i18n-with-log-report.pod

```diff
@@ -0,0 +1,206 @@
+=encoding utf8
+
+=head1 Implementing i18n in a Dancer application using Plugin::LogReport
+
+=head2 Introduction
+
+There are a number of ways to approach the internationalisation of a web
+application. I'm going to look at a method using L<Log::Report>, which includes
+very comprehensive translation functionality.
+
+More specifically, I will use its Dancer2 plugin,
+L<Dancer2::Plugin::LogReport>. If you haven't already, please look at my other
+article on that.
+
+=head2 A basic application
+
+Let's get started by creating a basic application with some strings that we want
+translated. We'll look at translating the content of templates later.
+
+ # In your route handler
+ package MyApp;
+ use Dancer2;
+ use Dancer2::Plugin::LogReport 'my-domain'; # domain used to group sets of messages
+ 
+ get '/' => sub {
+   notice "Hello world";
+   template 'index';
+ };
+
+ # In the index template
+ [% FOREACH message IN session.messages %]
+   <p>[% message.toString %]</p>
+ [% END %]
+
+By running this application, you should see C<Hello world> on the index page.
+
+=head2 Translating strings
+
+L<Log::Report> has L<a lot of functions to translate
+strings|Log::Report#Language-Translations>. We'll touch on 2 of the commonly
+used ones here.
+
+Firstly to simply translate a fixed string into another, we can use C<__> (2
```

Comment:
I think I'd just start with: To translate...

But I tend to be overly-terse so perhaps ignore me. :smile: